src/components/app/app.js

```diff
@@ -35,23 +40,55 @@ export class AppComponent {
       onAboutClick: this.handleAboutOnclick.bind(this)
     });
     this.toolBarComponent.mount();
-    this.drawerComponent = new DrawerComponent(this.drawerMountPoint, {
-      onAddAccountClick: this.handleAddAccountClick.bind(this),
-      onAddTagClick: this.handleAddTagOnclick.bind(this)
-    });
-    this.drawerComponent.mount();
     this.transactionsComponent = new TransactionsComponent(
       this.transactionsMountPoint
     );
     this.transactionsComponent.mount();
-    this.addAccountDialog = new AddAccountComponent(this.addAccountMountPoint);
+    this.addAccountDialog = new AddAccountComponent(this.addAccountMountPoint, {
+      onAddAccountConfirmed: this.handleAddAccountConfirmed.bind(this)
+    });
+
     this.addAccountDialog.mount();
     this.addTagDialog = new AddTagComponent(this.addTagMountPoint);
     this.addTagDialog.mount();
     this.aboutDialog = new AboutComponent(this.aboutMountPoint);
     this.aboutDialog.mount();
   }
 
+  getCurrentAccounts() {
+    StorageService.get("accounts").then(accounts => {
+      if (!accounts) {
+        accounts = [
+          new Account("Privat", "", "UAH"),
+          new Account("Payoneer", "", "USD")
+        ];
+        StorageService.set("accounts", accounts);
+      }
+      this.initDrawer(accounts);
+    });
+  }
+
+  initDrawer(accounts) {
+    this.drawerComponent = new DrawerComponent(this.drawerMountPoint, {
```

Comment:
do you need to redraw whole drawer? in my opinion it is better to refresh only accounts list